exercises/space-age/.meta/generator/space_age_cases.rb

```diff
@@ -0,0 +1,46 @@
+class SpaceAgeCase < ExerciseCase
+  SECONDS_PER_YEAR_ON_EARTH = 31557600
+
+  def workload
+    [
+      "age = SpaceAge.new(#{underscore_format(seconds)})",
+      indent(4, assertion),
+    ].join("\n")
+  end
```

Comment:
Now workload is the only public method as per the [docs](https://github.com/exercism/xruby#generated-test-suites).